### PR TITLE
Mizanur assignment

### PR DESCRIPTION
**Pull request by mizanur-007**

### CODE REVIEW

File: Rid-Assignment/question-4.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+number = int(input("Enter a number: "))
+
+if number % 2 == 0 and number % 5 == 0:
+    print("This number is divisible by 2 and 5.")
+elif number % 2 == 0:
+    print("The number is even")
```
Comment (noyonict) [inline Rid-Assignment/question-4.py]: Should be "[number] is even"

File: Rid-Assignment/question-4.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+else:
+    print("The number is odd")
```
Comment (noyonict) [inline Rid-Assignment/question-4.py]: same as above

File: Rid-Assignment/question-9.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+def count_even_numbers(numbers):
+    count = 0
+    for item in numbers:
+        if item % 2 ==0:
+            count = count + 1
```
Comment (noyonict) [inline Rid-Assignment/question-9.py]: count += 1

File: Rid-Assignment/question-10.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    for number in range(num1,num2):
+        if number >1:
+            for i in range(2,number):
+                if number % i == 0:
+                    break
+            else:
+                list.append(number)
+    return list
+
+number1 = int(input("Enter starting number: "))
```
Comment (noyonict) [inline Rid-Assignment/question-10.py]: variable name starting_number looks better than number1

File: Rid-Assignment/question-10.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+number2 = int(input("Enter ending number: "))
```
Comment (noyonict) [inline Rid-Assignment/question-10.py]: same as above

**Review by noyonict — APPROVED** [review]
looks okay

File: Rid-Assignment/question-4.py (hunk 1 above)
Comment (mizanur-007) [reply Rid-Assignment/question-4.py]: I have updated it

File: Rid-Assignment/question-4.py (hunk 2 above)
Comment (mizanur-007) [reply Rid-Assignment/question-4.py]: I have updated it

File: Rid-Assignment/question-9.py (hunk 3 above)
Comment (mizanur-007) [reply Rid-Assignment/question-9.py]: I have updated it

File: Rid-Assignment/question-10.py (hunk 4 above)
Comment (mizanur-007) [reply Rid-Assignment/question-10.py]: I have updated it

File: Rid-Assignment/question-10.py (hunk 5 above)
Comment (mizanur-007) [reply Rid-Assignment/question-10.py]: I have updated it

**Review by noyonict — APPROVED** [review]
looks okay

**Pull request merged**